.github/stale.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Stale issue handler"
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label / comment or this will be closed in 5 days'
+        stale-pr-message: 'This PR is stale because it has been open for 30 days with no activity. Remove stale label / comment or this will be closed in 5 days'
+        days-before-stale: 30
```

Comment:
30 days may be too aggressive for Twirp, what about 60? (we can change this later btw)

Reply:
60 sounds good